src/surge-fx/KnobSource.h

```diff
@@ -11,7 +11,8 @@
 
 struct ConcreteCM : sst::jucegui::data::ContinuousModulatable
 {
-    ConcreteCM() {};
+    ConcreteCM(SurgefxAudioProcessor &p, int i) : processor(p), id(i) {}
```

Comment:
If you make the base class here just continuous as opposed to continuous modiularanle you will get a better result